datalake/src/test/java/com/sequenceiq/datalake/service/sdx/SdxServiceTest.java

```diff
@@ -271,37 +278,49 @@ static Object[][] failedParamProvider() {
     static Object[][] razCloudPlatformDataProvider() {
         return new Object[][]{
                 // testCaseName cloudPlatform
-                {"CloudPlatform.AWS multiaz=true", CloudPlatform.AWS, true},
-                {"CloudPlatform.AWS multiaz=false", CloudPlatform.AWS, false},
-                {"CloudPlatform.AZURE multiaz=false", CloudPlatform.AZURE, false}
+                {"CloudPlatform.AWS multiaz=true", AWS, true},
+                {"CloudPlatform.AWS multiaz=false", AWS, false},
+                {"CloudPlatform.AZURE multiaz=false", AZURE, false},
+                {"CloudPlatform.GCP multiaz=false", GCP, false}
         };
     }
 
     static Object[][] razCloudPlatformAndRuntimeDataProvider() {
         return new Object[][]{
                 // testCaseName cloudPlatform runtime
-                {"CloudPlatform.AWS", CloudPlatform.AWS, "7.2.2"},
-                {"CloudPlatform.AZURE", CloudPlatform.AZURE, "7.2.1"}
+                {"CloudPlatform.AWS", AWS, "7.2.2"},
+                {"CloudPlatform.AZURE", AZURE, "7.2.2"},
```

Comment:
the required minimum versions do not match in the unit test either

Reply:
yes. since we have updated min runtime to 7.2.2 for AWS as part of this change. 7.2.1 is very old and it should not be used.